src/components/appHeader/apiCalls.js

```diff
@@ -25,6 +34,9 @@ export const handleLogin = (e, email, password, history, setErrors) => {
                 .then(json => {
```

Comment:
move this to after the .then for getUserInfo. We will add in more checking here later to make sure responses come back successful, etc and don't want to redirect if there's an error.